common/test/acceptance/pages/lms/edxnotes.py

```diff
@@ -523,7 +523,7 @@ def text(self):
         Returns text of the note.
         """
         self.show()
-        element = self.q(css=self._bounded_selector(".annotator-annotation > div"))
+        element = self.q(css=self._bounded_selector(".annotator-annotation > div.annotator-note"))
```

Comment:
I think it is just making it clear what the structure is within the annotation viewer. I don't feel particularly strongly about it, but also don't see a reason to change it.